Scripts/DCS-BIOS/lib/common/common.lua

```diff
@@ -0,0 +1,8 @@
+function GetJSON()
+    local json = loadfile([[Scripts\JSON.lua]])
+    if(json == nil) then
+        json = loadfile(DCS_INSTALLATION_ROOT.."\\Scripts"..[[\JSON.lua]])
```

Comment:
I've been thinking about this more and I think we definitely want to decouple this from having dcs installed on the machine, that way we can run this in github actions

Reply:
This is now an external file where user specifies its DCS World installation location.

Reply:
Yes let's do that but meanwhile get this merged. I don't know shit about Github Actions and I don't want this hanging out here since the rate of change in DCS-BIOS is close to the Argentinian inflation.